docs/upgrade/v1-0-3-to-v1-1-0.md

```diff
@@ -6,6 +6,32 @@ title: ""
 
 # Upgrade from v1.0.3 to v1.1.0
 
+## Warning
+
+Please do not upgrade a running cluster to v1.1.0 if the cluster has the following configuration:
+
+* The harvester-mgmt network contains two or more network interfaces.
+```
+  networks:
+    harvester-mgmt:
+      interfaces:
+      - name: ens5
+      - name: ens6
+      method: dhcp
+```
+
+* Defining a cluster_network in the configuration file with harvester-mgmt network:
+```
+cluster_networks:
+  vlan:
+    enable: true
+    description: "harvester-mgmt"
+    config:
+      defaultPhysicalNIC: harvester-mgmt
+```
+
+Related issue: [[BUG] Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface](https://github.com/harvester/harvester/issues/3045)
```

Comment:
can we add `:::caution` to this section, thanks.

Reply:
Update to `:::warning` block. Thanks